robottelo/constants.py

```diff
@@ -837,7 +837,7 @@
         'skip_for_ui': True},
     {'name': 'HostCollection', 'controller': 'katello_host_collections'},
     {'name': 'Architecture', 'controller': 'architectures'},
-    {'name': 'HardwareModel', 'controller': 'models'},
+    {'name': 'HardwareModel', 'controller': 'models', 'skip_for_ui': True},
```

Comment:
I think the skip logic should be implemented on the uitests themselves, something like create a copy of this constant and then pop the value for HardwareModel in order to not pick it up.